app/src/main/java/app/revanced/integrations/shared/patches/SpoofSignaturePatch.java

```diff
@@ -0,0 +1,69 @@
+package app.revanced.integrations.shared.patches;
+
+import android.content.Context;
+import android.content.pm.PackageManager;
+
+import org.apache.commons.lang3.StringUtils;
+
+import app.revanced.integrations.shared.settings.BaseSettings;
+import app.revanced.integrations.shared.utils.Logger;
+
+@SuppressWarnings("unused")
+public class SpoofSignaturePatch extends GmsCoreSupport {
+
+    /**
+     * Injection point.
+     */
+    public static String spoofPackageName(Context context) {
+
+        // Package name of ReVanced.
+        final String packageName = context.getPackageName();
+
+        if (!BaseSettings.SPOOF_SIGNATURE.get()) {
+            return packageName;
+        }
+
+        final PackageManager packageManager = context.getPackageManager();
+
+        // Package name of YouTube or YouTube Music.
+        String originalPackageName;
+
+        try {
+            originalPackageName = packageManager
+                    .getPackageInfo(packageName, PackageManager.GET_META_DATA)
```

Comment:
uhm this function should be `getSpoofedPackageName`

Reply:
This has the same functionality as `PackageSpoofUtils.spoofPackageName`
(integrations cannot access GmsCore's `PackageSpoofUtils.spoofPackageName`)